don't hardcode path to basename - Pablo Escobar (sciCORE, SIB, UniBAS)

diff -ru ncbi-blast-2.2.30+-src.orig/c++/src/build-system/Makefile.in.top ncbi-blast-2.2.30+-src/c++/src/build-system/Makefile.in.top
--- ncbi-blast-2.2.30+-src.orig/c++/src/build-system/Makefile.in.top	2014-08-01 20:43:59.000000000 +0200
+++ ncbi-blast-2.2.30+-src/c++/src/build-system/Makefile.in.top	2015-03-10 15:11:09.140548221 +0100
@@ -46,7 +46,7 @@
 	-rm -f $(libdir)/lib*-static.a
 	cd $(libdir)  && \
 	    for x in *.a; do \
-	        $(LN_S) "$$x" "`/usr/bin/basename \"$$x\" .a`-static.a"; \
+	        $(LN_S) "$$x" "`basename \"$$x\" .a`-static.a"; \
 	    done
 	cd $(includedir0) && find * -name CVS -prune -o -print |\
             cpio -pd $(pincludedir)
